alerts.tf

```diff
@@ -25,8 +25,8 @@ module "div-server-errors-alert" {
   app_insights_name = "div-${var.env}"
 
   alert_name = "div-server-errors"
-  alert_desc = "Found HTTP requests with 400 error response code (bad request) in div-${var.env}."
-  app_insights_query = "requests | where resultCode startswith \"5\"  | where name !contains \"/health\""
+  alert_desc = "Found HTTP requests with 500 error response code (bad request) in div-${var.env}."
+  app_insights_query = "requests | where resultCode startswith '5'  | where name !contains '/health' | where datetime_diff('hour', timestamp, startofday(timestamp)) !between (2 .. 5)"
```

Comment:
The nightly job runs between 2-4.30 am, so we check if the hour fits between these hours